wear/src/main/java/io/homeassistant/companion/android/sensors/SensorReceiver.kt

```diff
@@ -0,0 +1,71 @@
+package io.homeassistant.companion.android.sensors
+
+import android.annotation.SuppressLint
+import dagger.hilt.android.AndroidEntryPoint
+import io.homeassistant.companion.android.BuildConfig
+import io.homeassistant.companion.android.common.sensors.BatterySensorManager
+import io.homeassistant.companion.android.common.sensors.SensorManager
+import io.homeassistant.companion.android.common.sensors.SensorReceiverBase
+
+@AndroidEntryPoint
+class SensorReceiver : SensorReceiverBase() {
+
+    override val tag: String
+        get() = TAG
+
+    override val currentAppVersion: String
+        get() = BuildConfig.VERSION_NAME
+
+    override val managers: List<SensorManager>
+        get() = MANAGERS
+
+    companion object {
+        const val TAG = "SensorReceiver"
+        val MANAGERS = listOf(
+//            ActivitySensorManager(),
+//            AppSensorManager(),
+//            AudioSensorManager(),
+            BatterySensorManager(),
+//            BluetoothSensorManager(),
+//            DNDSensorManager(),
+//            GeocodeSensorManager(),
+//            KeyguardSensorManager(),
+//            LastRebootSensorManager(),
+//            LastUpdateManager(),
+//            LightSensorManager(),
+//            LocationSensorManager(),
+//            MobileDataManager(),
+//            NetworkSensorManager(),
+//            NextAlarmManager(),
+//            NotificationSensorManager(),
+//            PhoneStateSensorManager(),
+//            PowerSensorManager(),
+//            PressureSensorManager(),
+//            ProximitySensorManager(),
+//            StepsSensorManager(),
+//            StorageSensorManager(),
+//            TimeZoneManager(),
+//            TrafficStatsManager()
+        )
+
+        const val ACTION_REQUEST_SENSORS_UPDATE =
+            "io.homeassistant.companion.android.background.REQUEST_SENSORS_UPDATE"
+    }
+
+    // Suppress Lint because we only register for the receiver if the android version matches the intent
+    @SuppressLint("InlinedApi")
+    override val skippableActions = mapOf<String, String>()/*
+        "android.app.action.NEXT_ALARM_CLOCK_CHANGED" to NextAlarmManager.nextAlarm.id,
```

Comment:
For now lets remove these rather than comment out.

Reply:
Done